dev-python/openpyxl/openpyxl-2.6.1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
+inherit distutils-r1 vcs-snapshot
```

Comment:
Is vcs-snapshot still needed? Cant download the archive currently and test, but to me it looks like it's in standard form. If you can drop vcs-snapshot, then the EAPI can be bumped to 7 as well with this version bump. Looks good otherwise!